tests/python/rest_api/test_jobs.py

```diff
@@ -83,7 +83,7 @@ def test_admin_get_job(self, jobs, tasks, org):
         for job in jobs[:8]:
             self._test_get_job_200("admin2", job["id"], job, **kwargs)
 
-    @pytest.mark.parametrize("org_id", ["", None, 1, 2])
+    @pytest.mark.parametrize("org_id", [""])
```

Comment:
Please use the same set of orgs here.

Reply:
Need to be sure that our test system covers primary use cases connected with changes in the PR.

Reply:
Fixed